doc/api-reference/openquake.hazardlib.gsim.rst

```diff
@@ -423,6 +423,14 @@ campbell_2003
     :undoc-members:
     :show-inheritance:
 
+campbell_bozorgnia_2003_world
+-------------------------------------------------------
+
+.. automodule:: openquake.hazardlib.campbell_bozorgnia_2003_world
```

Comment:
The documentation tests are broken because here you must change
 openquake.hazardlib.campbell_bozorgnia_2003_world ->  openquake.hazardlib.gsim.campbell_bozorgnia_2003_world